src/distances.jl

```diff
@@ -24,64 +84,45 @@ julia> Ellipsoidal([1.0,0.5], [π/2])
 julia> Ellipsoidal([1.0,0.5,0.5], [π/2,0.0,0.0])
 ```
 """
+
 struct Ellipsoidal{N,T} <: Metric
```

Comment:
As is, this contains an abstractly typed field. I think it should be
```julia
struct Ellipsoidal{N,M<:Mahalanobis} <: Metric
    dist::M
...
end
```

Reply:
Thanks for catching this @dkarrsch, very good point. @rmcaixeta could you fix it? This will also retrigger the build for us.

Reply:
Thanks @dkarrasch. Updating here in some minutes